assets/src/edit-story/types.js

```diff
@@ -196,8 +196,11 @@ StoryPropTypes.elements.text = PropTypes.shape({
   color: PatternPropType.isRequired,
   backgroundTextMode: PropTypes.oneOf(Object.values(BACKGROUND_TEXT_MODE)),
   backgroundColor: PatternPropType,
-  fontFamily: PropTypes.string,
-  fontFallback: PropTypes.array,
+  font: PropTypes.shape({
```

Comment:
Let's do a top-level const in this module as well that we can use in other components' proptypes. I think I made a note on that above.